kiali-server/values.yaml

```
@@ -113,8 +113,10 @@ login_token:
 
 server:
   port: 20001
+  nodePort: 32475
```

I also just noticed this is in camelCase. All settings within values and the Kiali CR are listed in snake_case. For consistency, this should be `node_port`:

```suggestion
  node_port: 32475
```